content/en/user-guide/aws/bedrock/index.md

```diff
@@ -75,5 +86,5 @@ $ awslocal bedrock-runtime converse \
 
 ## Limitations
 
-* LocalStack Bedrock implementation is mock-only and does not run any LLM model locally.
+* LocalStack Bedrock currently only officially supports text-based models.
```

Comment:
What does "currently only officially" mean exactly?

Reply:
That at this point in time (=currently) we only officially support text-based models as opposed to image or other kinds of binary data models

Reply:
My question would be what kind of models are unofficially supported :sweat_smile: . It's fine for me now, this was just the first question popping in my head. For me they are either supported or not, but this phrasing leads me to believe other models might be unofficially supported :P